stock_barcodes/wizard/stock_barcodes_read.py

```diff
@@ -688,12 +688,14 @@ def reset_qty(self):
 
     def open_actions(self):
         self.display_menu = True
-        return self.env.ref(
-            "stock_barcodes.action_stock_barcodes_action_client"
-        ).read()[0]
+        return (
+            self.env.ref("stock_barcodes.action_stock_barcodes_action_client")
+            .sudo()
+            .read()[0]
+        )
```

Comment:
Instead of this, use `_for_xml_id`, that is specially done for these cases:

```suggestion
        return self.env["ir.actions.act_window"]._for_xml_id(
            "stock_barcodes.action_stock_barcodes_action_client"
        )
```

and the same for the other.